MIDIchlorians/MIDIchlorians/Info.plist

```diff
@@ -14,12 +14,30 @@
 	<string>$(PRODUCT_NAME)</string>
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>db-8a5t7h2dpsfjunc</string>
```

Comment:
when you merge, please squash into a single commit so this key doesn't exist in an easily discoverable manner